lib/intelligent_foods.rb

```diff
@@ -12,8 +12,13 @@
 require "intelligent_foods/errors"
 require "intelligent_foods/resources/object"
 require "intelligent_foods/resources/order"
+require "intelligent_foods/resources/order_item"
+require "intelligent_foods/serializers/order_item_serializer"
```

Comment:
nitpicking: can we organize these so theyre grouped? load all serializers first, then resources or something? Just thinking about ease of reading.